CONTRIBUTING.md

```diff
@@ -234,3 +234,21 @@ ninja -C build git-hooks
 
 Either of the above commands will ensure that Git hooks are properly setup for
 the project.
+
+### Code Coverage
+
+HSE uses gcovr to get code coverage. To enable coverage and get the coverage
+reports run the following.
+
+```shell
+meson build --buildtype=debug -Db_coverage=true
```

Comment:
Use `builddir` instead of `build` since I think that is what we use elsewhere.

Reply:
Seems to be build throughout the contributing file.